akka-typed/src/main/scala/akka/typed/javadsl/Actor.scala

```diff
@@ -58,44 +55,19 @@ object Actor {
    * @see [[Actor#mutable]]
    */
   abstract class MutableBehavior[T] extends ExtensibleBehavior[T] {
-    @throws(classOf[Exception])
-    override final def receiveMessage(ctx: akka.typed.ActorContext[T], msg: T): Behavior[T] =
-      onMessage(msg)
+    private val receive: Receive[T] = createReceive
```

Comment:
Added a unittest to show that this was indeed a problem. Solved it for now by making `receive` a `lazy val` - I know that this is not without overhead but I can't quite think of a faster alternative. Ideas?